### PR TITLE
Add support for regulatory bundles

### PR DESCRIPTION
**Pull request by macleanpinto**
# Example query results
<details>
  <summary>Results</summary>
  
```
> select sid,account_sid, friendly_name from twilio_account_bundle limit 2;
+------------------------------------+------------------------------------+-------------------------------------------------------+
| sid                                | account_sid                        | friendly_name                                         |
+------------------------------------+------------------------------------+-------------------------------------------------------+
| BU6260dd03ec30babfaaab95b0a41039d9 | AC62f7582bceded11b89df39377bdebe85 | Maclean Pinto Algeria National 2021-10-06 at 16:43:46 |
| BU628c58cc00ecad0d76071ede1df30876 | AC62f7582bceded11b89df39377bdebe85 | test01                                                |
+------------------------------------+------------------------------------+-------------------------------------------------------+

```
</details>

### CODE REVIEW

**Comment by rajlearner17** [comment]
@macleanpinto Thanks for using Steampipe!! Great to see this contribution 👍 . We will check this and revert back.

File: docs/tables/twilio_account_bundles.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+select 
+  sid
+  account_sid, 
+  friendly_name, 
+  email, 
+  regulation_sid, 
+  url, 
+  valid_until 
+from 
+  twilio.twilio_account_bundles;
```
Comment (bigdatasourav) [inline docs/tables/twilio_account_bundles.md:21]:
```suggestion
  twilio_account_bundles;
```

File: docs/tables/twilio_account_bundles.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+```
+
+### List bundles in draft status
+
+```sql
+select 
+  sid
+  account_sid, 
+  friendly_name, 
+  email,
```
Comment (bigdatasourav) [inline docs/tables/twilio_account_bundles.md:31]: please update the query format as below -
```
select 
  sid
  account_sid, 
  friendly_name, 
  email, 
  regulation_sid, 
  url, 
  valid_until 
  from
  twilio_account_bundles 
  where
  status ='draft';
```

File: twilio/plugin.go [hunk 3]
```diff
@@ -34,6 +34,7 @@ func Plugin(ctx context.Context) *plugin.Plugin {
 			"twilio_messaging_service":             tableTwilioMessagingService(ctx),
 			"twilio_serverless_service":            tableTwilioServerlessService(ctx),
 			"twilio_serverless_service_function":   tableTwilioServerlessServiceFunction(ctx),
+			"twilio_account_bundles":               tableTwilioAccountBundles(ctx),
```
Comment (bigdatasourav) [inline twilio/plugin.go:37]: table names should in alphabetically sorted order

File: twilio/table_twilio_bundles.go [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "account_sid",
+				Description: "The SID of the Account that created the Bundle resource.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "sid",
+				Description: "The unique string that we created to identify the Bundle resource.",
```
Comment (bigdatasourav) [inline twilio/table_twilio_bundles.go:28]: Please update the column in below order for better presentation - 
```
#top 3-4 should be primary columns like (id,name,status, create date)
account_sid
sid
friendly_name
date_created
status

#rest should be in sorted order
date_updated
email
regulation_sid
status_callback
url
valid_until

#json columns if any should be in last, those also should be in sorted order
```

File: docs/tables/twilio_account_bundles.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
+# Table: twilio_account_bundles
```
Comment (bigdatasourav) [inline docs/tables/twilio_account_bundles.md:1]:
```suggestion
# Table: twilio_account_bundle
```

File: twilio/table_twilio_account_bundles.go [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+	openapi "github.com/twilio/twilio-go/rest/numbers/v2"
+)
+
+func tableTwilioAccountBundles(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "twilio_account_bundles",
```
Comment (bigdatasourav) [inline twilio/table_twilio_account_bundles.go:15]: Please update the table name as `twilio_account_bundle`, we maintain our table names in singular format. 
Also please update the file names as well.

File: twilio/table_twilio_account_bundles.go [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "url",
+				Description: "The absolute URL of the Bundle resource.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "valid_until",
+				Description: "The date and time when the resource will be valid until. ",
```
Comment (bigdatasourav) [inline twilio/table_twilio_account_bundles.go:75]:
```suggestion
				Description: "The date and time when the resource will be valid until.",
```

File: twilio/table_twilio_account_bundles.go [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+	limit := d.QueryContext.Limit
+	if d.QueryContext.Limit != nil {
+		if int(*limit) < maxResult {
+			maxResult = int(*limit)
+		}
+	}
+	req.SetLimit(maxResult)
+
+	resp, err := client.NumbersV2.StreamBundle(req)
+	fmt.Println(resp)
```
Comment (bigdatasourav) [inline twilio/table_twilio_account_bundles.go:104]:
```suggestion
```

File: docs/tables/twilio_account_bundles.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+select 
+  sid
+  account_sid, 
+  friendly_name, 
+  email, 
+  regulation_sid, 
+  url, 
+  valid_until 
+from 
+  twilio_account_bundles;
```
Comment (bigdatasourav) [inline docs/tables/twilio_account_bundles.md:21]:
```suggestion
  twilio_account_bundle;
```

File: docs/tables/twilio_account_bundles.md [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
```
Comment (bigdatasourav) [inline docs/tables/twilio_account_bundles.md:36]:
```suggestion
  twilio_account_bundle
```

File: docs/tables/twilio_account_bundles.md [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+where 
+  status ='draft';
```
Comment (bigdatasourav) [inline docs/tables/twilio_account_bundles.md:38]:
```suggestion
  status = 'draft';
```

File: twilio/plugin.go [hunk 12]
```diff
@@ -25,6 +25,7 @@ func Plugin(ctx context.Context) *plugin.Plugin {
 			"twilio_account":                       tableTwilioAccount(ctx),
 			"twilio_account_address":               tableTwilioAccountAddress(ctx),
 			"twilio_account_application":           tableTwilioAccountApplication(ctx),
+			"twilio_account_bundles":               tableTwilioAccountBundles(ctx),
```
Comment (bigdatasourav) [inline twilio/plugin.go:28]:
```suggestion
			"twilio_account_bundle":               tableTwilioAccountBundle(ctx),
```

File: twilio/table_twilio_account_bundles.go [hunk 13]
```diff
@@ -0,0 +1,122 @@
...
+	"context"
+	"fmt"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+	openapi "github.com/twilio/twilio-go/rest/numbers/v2"
+)
+
+func tableTwilioAccountBundles(ctx context.Context) *plugin.Table {
```
Comment (bigdatasourav) [inline twilio/table_twilio_account_bundles.go:13]:
```suggestion
func tableTwilioAccountBundle(ctx context.Context) *plugin.Table {
```

**Review by bigdatasourav — APPROVED** [review]
LGTM

**Pull request merged**